seedDB/README.md

```diff
@@ -1,17 +1,20 @@
-These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+# cBioPortal Seed Database
 
-- DB schema version **1.3.0**
+These files are MySQL database dump files for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+
+- DB schema version **2.1.0**
 
 Which is suitable for:
 
-- release version **1.3.1** of cBioPortal.  
-- release version **> 1.3.1** of cBioPortal (:warning: with possible migration script step). 
+- release version **1.5.0** of cBioPortal.  
```

Comment:
I also tested loading a public study using SKCM study (https://github.com/cBioPortal/datahub/blob/master/public/skcm_tcga.tar.gz). It loaded fine :+1: 

Some screenshots:
![image](https://cloud.githubusercontent.com/assets/2900303/25529896/58a41118-2c24-11e7-8e3b-952de30ad1a0.png)

![image](https://cloud.githubusercontent.com/assets/2900303/25529910/6b88ef56-2c24-11e7-85c1-648023fc4cca.png)

![image](https://cloud.githubusercontent.com/assets/2900303/25529949/97b8afe4-2c24-11e7-8c03-2a554d8a2dd9.png)